src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/correspondence/consentorder/ContestedDraftOrderCorresponder.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.correspondence.consentorder;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.wrapper.SolicitorCaseDataKeysWrapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.NotificationService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.correspondence.CaseDetailsEmailOnlyAllSolicitorsCorresponder;
+
+@Component
+@Slf4j
+public class ContestedDraftOrderCorresponder extends CaseDetailsEmailOnlyAllSolicitorsCorresponder {
+
+    @Autowired
+    public ContestedDraftOrderCorresponder(NotificationService notificationService) {
+        super(notificationService);
+    }
+
+    @Override
+    protected void emailApplicantSolicitor(CaseDetails caseDetails) {
+        if (notificationService.isApplicantSolicitorResponsibleToDraftOrder(caseDetails.getData())
+            && notificationService.isApplicantSolicitorAgreeToReceiveEmails(caseDetails)) {
```

Comment:
This check should not be valid anymore  - we just checking digital or nondigital. In this method still check consent.